DEVELOPMENT.md

```diff
@@ -140,7 +140,6 @@ by [`Task`](docs/Concepts.md#task)):
 
 ```shell
 ko apply -f config/
```

Comment:
can we also remove the description above that line that says `, including knative/build (which is wrapped by Task):`

Reply:
Sure.